Components/LandingNav.tsx

```diff
@@ -2,19 +2,19 @@ import Link from "next/link";
 import { useRouter } from "next/router";
 import { DarkModeToggle } from "../Components/components";
 
-const LandingNav = () => {
+const LandingNav = ({ error404Check }: { error404Check: boolean }) => {
```

Comment:
```suggestion
const LandingNav = ({ pageNotFound }: { pageNotFound: boolean }) => {
```

change the `prop` name to `pageNotFound` instead of `error404Check`. And if this page is going to show when there is any error or page crash in the application then change it to `errorOccurred` or something meaningful in that case.

Reply:
I have changed the prop name as requested as well as the boolean logic sir.
I don't understand the 'errorChanged' request sir, can you please explain it once more?